service/src/main/resources/api/service_openapi.yaml

```diff
@@ -516,6 +516,9 @@ components:
             A version of the landing zone. If not set the most recent will be used.
             If two versions available - 'v1' and 'v2' then 'v2' will be selected.
           type: string
+        region:
+          description: The region where the landing zone is deployed.
+          type: string
```

Comment:
Will we be backfilling regions for existing landing zones? If not, I think the documentation should call out that the region will be "" for landing zones created prior to some date.